src/Operation/Delete.php

```diff
@@ -43,8 +43,7 @@
  */
 class Delete implements Executable, Explainable
 {
-    /** @var integer */
-    private static $wireVersionForHint = 9;
+    private const WIRE_VERSION_FOR_HINT = 5;
```

Comment:
```suggestion
    private const WIRE_VERSION_FOR_HINT = 9;
```

Not sure how this was changed from 9 to 5, but the correct wire version is definitely 9. See: https://github.com/mongodb/specifications/blob/master/source/wireversion-featurelist.rst